src/AppBundle/Form/PostType.php

```diff
@@ -43,13 +43,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'attr' => array('autofocus' => true),
                 'label' => 'label.title',
             ))
-            ->add('summary', 'textarea', array('label' => 'label.summary'))
-            ->add('content', 'textarea', array(
+            ->add('summary', 'Symfony\Component\Form\Extension\Core\Type\TextareaType', array('label' => 'label.summary'))
```

Comment:
This `summary` field is a `string` type, why use textarea? or perhaps we should redefine it as `text`?